cmd/buildah/rename.go

```diff
@@ -45,6 +45,16 @@ func renameCmd(c *cli.Context) error {
 		return errors.Errorf("renaming a container with the same name as its current name")
 	}
 
+	builders, err := openBuilders(store)
```

Comment:
Why not use?

			if _, err := openBuilder(getContext(), store, name); err == nil {
                                    return errors.Errorf("The container name %q is already in use by container %q", newName, build.ContainerID) 
}

Reply:
updated, thanks.